source/release-notes/backend.txt

```diff
@@ -13,6 +13,33 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240417:
+
+17 April 2024 Release
+~~~~~~~~~~~~~~~~~~~~~
+
+- Fixed Google Cloud Provider ``us-central1`` label to read "Iowa" instead of "Ohio" in App Services Deployment
+  Region settings.
+
+- Improved user caching for authentication provider user metadata. For more information, refer to
+  notes in :ref:`Authentication Provider Metadata <auth-provider-metadata>`.
+
+- There are now three types of private endpoints: general, sync, and legacy. Legacy endpoints must migrate
```

Comment:
Do we ever use anything else in the RNs to indicate breaking changes or when action may be required for users (e.g. formatting, section, admonitions)?

Reply:
Other than UI Improvements, I don't see any former labels. We could be flexible in including more specific labels (bug fixes, breaking changes) if there are enough of the type in the notes to warrant it!